demo/allegro5/nuklear_allegro5.h

```diff
@@ -102,10 +105,12 @@ nk_allegro5_font_get_text_width(nk_handle handle, float height, const char *text
        as nuklear uses variable size buffers and al_get_text_width doesn't
        accept a length, it infers length from null-termination
        (which is unsafe API design by allegro devs!) */
-    char strcpy[len+1];
-    strncpy((char*)&strcpy, text, len);
+    strcpy = malloc(len + 1);
```

Comment:
Maybe it would be better to just make this function take a null terminated string and have the caller handle the memory. This would allow the caller to avoid allocations by reusing the buffer if they want to do so.